banners/desktop/content/BannerSlides.vue

```diff
@@ -11,7 +11,7 @@
 	</KeenSliderSlide>
 	<KeenSliderSlide :is-current="currentSlide === 1">
 		<p>
-			<span class="wmde-banner-text-animated-highlight">{{ visitorsVsDonorsSentence }}</span>
+			<span class="wmde-banner-slider-text-animated-highlight">{{ visitorsVsDonorsSentence }}</span>
```

Comment:
The class selector here and in the other slides was purposely changed to `wmde-banner-text-animated-highlight` in order to match the one in the message highlighting. It prevents the constant copy/pasting errors we had in the last campaign.

These changes should be reverted and the selectors in the Dandy tests updated instead.